### DISCUSSION TITLE
remoteapps epoptes doesn't work

### DISCUSSION THREAD

**Discussion by kais-iae** [epoptes/epoptes#93] [Q&A]
Hi,

Using the following document:http://www.epoptes.org/documentation/fat-clients
I could configure my epoptes on LTSP server but when i run "~$ remoteapps dbus lunch epoptes" on the fat client ternimal's nothing is happening.So i would like to know how i can resolve this problem ?

**Comment by alkisg** [comment]
Hi, it sounds like you didn't configure LTSP correctly.
It doesn't sound like a problem in the Epoptes code.

https://bugs.launchpad.net/ltsp/+bug/1811993
